admin/src/main/java/org/bahmni/module/admin/csv/service/CSVPatientService.java

```diff
@@ -74,20 +74,22 @@ public Patient save(PatientRow patientRow) throws ParseException {
     private void addPersonAttributes(Patient patient, PatientRow patientRow) throws ParseException  {
         for (KeyValue attribute : patientRow.attributes) {
             PersonAttributeType personAttributeType = findAttributeType(attribute.getKey());
-            if (personAttributeType.getFormat().equalsIgnoreCase("org.openmrs.Concept")) {
-                Concept concept = getConceptByName(attribute.getValue());
-                if (concept != null) {
-                    patient.addAttribute(new PersonAttribute(personAttributeType, concept.getId().toString()));
-                } else {
-                    throw new RuntimeException("Invalid value for Attribute." + attribute.getKey());
-                }
-            } else if (personAttributeType.getFormat().startsWith("java.lang.")) {
-                patient.addAttribute(new PersonAttribute(findAttributeType(attribute.getKey()), attribute.getValue()));
-           } else if (personAttributeType.getFormat().startsWith("org.openmrs.util.AttributableDate")) {
-                //Validating the Date format
-                String dateString = attribute.getValue();
-                getDateFromString(dateString);
-                patient.addAttribute(new PersonAttribute(findAttributeType(attribute.getKey()),dateString));
+            if(!StringUtils.isBlank(attribute.getValue())){
```

Comment:
This will possibly rule out having blank string (empty string) ever as value for any attribute type. Which i think is fine, considering OpenMRS does not create a EAV entry in the database, if no attrib value is specified. 

However, can we write this as an gaurd clause? to avoid nested conditions
e.g. 
if (isBlank(attribute.getValue())) continue;